### DISCUSSION TITLE
Oracle Table Compression Example in SQLAlchemy Oracle Document does not work.

### DISCUSSION THREAD

**Discussion by ybjeon01** [sqlalchemy/sqlalchemy#12075] [Usage Questions] [answered]
In SQLAlchemy's documentation, there is an example of table creation using compression, as seen here: [SQLAlchemy Oracle Dialect - Table Options](https://docs.sqlalchemy.org/en/20/dialects/oracle.html#oracle-table-options). I tested this example on Oracle 23, but the table creation failed.

```python
Table('mytable', metadata, Column('data', String(32)),
     oracle_compress=6)
```

According to Oracle’s official documentation, compression settings for tables do not accept numeric values. For reference, please see [Oracle Database Documentation - CREATE TABLE](https://docs.oracle.com/cd/E18283_01/server.112/e17118/statements_7002.htm#i2129627).

If the example is a mistake, I would be happy to submit a pull request with a different example that use correct argument.

**Comment by CaselIT** [comment, marked as answer]
Hi,

It's likely an error in the example. If you fix it in a PR it would be great!
